migrations/versions/000b9376b87b_.py

```diff
@@ -0,0 +1,42 @@
+"""empty message
+
+Revision ID: 000b9376b87b
+Revises: 6440077182f0
+Create Date: 2018-07-05 09:52:56.681093
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = '000b9376b87b'
+down_revision = '6440077182f0'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('event_locations',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(), nullable=False),
+    sa.Column('slug', sa.String(), nullable=False),
+    sa.PrimaryKeyConstraint('id'),
+    sa.UniqueConstraint('slug')
+    )
+    op.add_column('session_types', sa.Column('deleted_at', sa.DateTime(timezone=True), nullable=True))
+    op.add_column('social_links', sa.Column('deleted_at', sa.DateTime(timezone=True), nullable=True))
+    op.add_column('stripe_authorizations', sa.Column('deleted_at', sa.DateTime(timezone=True), nullable=True))
```

Comment:
This shouldn't be here.

Reply:
Isn't it generated automatically? What do you mean?

Reply:
Yeah, there are other changes as well.

Reply:
Maybe someone forgot to add these columns to the migration or maybe schedutron is not up to date with the migrations

Reply:
@srv-twry @Kreijstal Were these `deleted_at` fields added in some other migration file?